index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Bartender Pirate!</title>
+    <meta name="description" content="Arr! How you like yer drink, matey? A pirate bartender app.">
+    <meta name="author" content="Row and Yoli" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <link href="https://necolas.github.io/normalize.css/4.1.1/normalize.css" rel="stylesheet" type="text/css">
+    <link href="style.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+	<div class="binding-box">
```

Comment:
Possibly use a form tag instead of a div tag to follow HTML5 semantics and improve SEO.